client/src/components/AppointmentCreateForm.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+
+import "react-datetime/css/react-datetime.css";
+
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import Datetime from "react-datetime";
+import { FETCH_APPOINTMENTS } from "./PickupList";
+import { useHistory } from "react-router-dom";
+
+const CREATE_APPOINTMENT = gql`
+  mutation createAppointment($startTime:Int! $endTime:Int! $address:String! $note:String! ){
+    createAppointment(startTime:$startTime, endTime:$endTime, address:$address, note:$note){
+      id,
+      location{
+        id
+        address
+        note
+      },
+      startTime,
+      endTime
+  }}
+`;
+
+const AppointmentCreateForm: () => React$Node = () => {
+
+  const history = useHistory();
+  const backToLanding = () => {
+    history.push("/");
+  };
+
+  const [newFromTime, setNewFromTime] = useState('');
+  const [newToTime, setNewToTime] = useState('');
+  const [newAddress, setNewAddress] = useState('');
+  const [newNote, setNewNote] = useState(' ');
+
+  const [ createAppointment,
+          {loading:mutationLoading, error: mutationError},
+   ] = useMutation(CREATE_APPOINTMENT, {
+    refetchQueries: [{query: FETCH_APPOINTMENTS}],
+    onCompleted: () => backToLanding()
+  });
+
+  const fromTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupFromTime",
+    required: "required"
+  };
+  const toTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupToTime",
+    required: "required"
+  };
+  
+  var startTime = new Date(newFromTime);
+  var endTime = new Date(newToTime);
+
+  const handleSubmit = () => {
+    if (startTime>=endTime){
+        return [false, alert("Oops, 'End Time' must happen after 'Start time'! \n Please try again.")]
+    }else{
```

Comment:
I don't think you have the prettier plugin installed in VSCode because otherwise that plugin would have formatted this for you (usually there are spaces around `else`).

`prettier` should already be part of the dev dependencies (you can see it in our `package.json` file), and look at [this guide](https://prettier.io/docs/en/install.html) to learn how to run it from the terminal, and then [install the prettier VSCode plugin](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

The end result is that every time you save a javascript file, prettier should automatically trigger and formats the file for you.

Reply:
I am sure I have 'prettier' in my code. the automatic formatting thing happens in other places, like when I import some components or dependencies on the top few lines of the a file, 'prettier' helps to reorder names alphabetically. But it does not work here.

Reply:
sorting dependencies is not from `prettier`. I just tried your file and my VSCode auto formatted the file for me. You need to set up prettier on your VSCode so it autoformats.

Reply:
Ok, I just solved it on my local file, you will see it from the new commit soon. Thanks for reminding me that.